lib/mix/tasks/nimble.phx.gen.template.ex

```diff
@@ -0,0 +1,44 @@
+defmodule Mix.Tasks.Nimble.Phx.Gen.Template do
+  @shortdoc "Generates Nimble's Phoenix template"
+
+  use Mix.Task
+
+  alias Nimble.Phx.Gen.Template.{Project, Template}
+
+  @version Mix.Project.config()[:version]
+  @switches [api: :boolean, web: :boolean]
+
+  def run([version]) when version in ~w(-v --version) do
+    Mix.shell().info("Nimble.Phx.Gen.Template v#{@version}")
+  end
+
+  def run(args) do
+    if Mix.Project.umbrella?() do
+      Mix.raise("mix nimble.phx.gen.template can only be run inside an application directory")
+    end
+
+    {opts, _params} = parse_opts(args)
+
+    %Project{
+      is_api_project: opts[:api],
```

Comment:
FYI Elixir supports question marks in keys: 

```elixir
is_api_project?: opts[:api],
```

So we might use it (or not) 🤷